playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -93,17 +132,19 @@
   hosts: oo_masters_to_config
   roles:
   - role: etcd_migrate
+    r_etcd_migrate_action: add_ttls
```

Comment:
What about to remove `delegate_to` from form the `add_ttls.yml` and move the action to its own play:
```yaml
- name: Add TTLs on the first master
  hosts: groups.oo_first_master[0]
  roles:
  - role: etcd_migrate
    r_etcd_migrate_action: add_ttls
    etcd_peer: "{{ hostvars[groups.oo_etcd_to_migrate.0].ansible_default_ipv4.address }}"
    etcd_url_scheme: "https"
    etcd_peer_url_scheme: "https"
    when: etcd_migration_failed | length == 0
```